foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClient.java

```diff
@@ -171,19 +220,21 @@ public OpenAiChatCompletionOutput chatCompletion(
    * Stream#parallel()} on this stream is not supported.
    *
    * @param prompt a text message.
-   * @return A stream of message deltas
+   * @return A stream of text chunks
    * @throws OpenAiClientException if the request fails or if the finish reason is content_filter
-   * @see #streamChatCompletionDeltas(OpenAiChatCompletionParameters)
+   * @see #streamChatCompletionDeltas(OpenAiChatCompletionRequest)
    */
   @Nonnull
   public Stream<String> streamChatCompletion(@Nonnull final String prompt)
       throws OpenAiClientException {
```

Comment:
Why did you keep `streamChatCompletion(String)` with a behaviour change, but you deprecated `chatCompletion(String)` instead of doing the same behaviour change. It would be less breaking to also keep it,

Reply:
No behavior change
One method returns (stable) String, the other method returns deprecated class.

Reply:
I don't mind breaking changes.
`OpenAiClientTest.testThrowsOnContentFilter` was removed, I'm guessing this means `streamChatCompletion()` has a behaviour change.
If you had `chatCompletion(String)` not deprecated it could still exist but return the generated class.